DatadogObjc/Sources/RUM/RUM+objc.swift

```diff
@@ -378,7 +378,7 @@ public class DDRUMConfiguration: NSObject {
     }
 
     @objc public var appHangThreshold: TimeInterval {
-        set { swiftConfig.appHangThreshold = newValue }
+        set { swiftConfig.appHangThreshold = newValue == 0 ? nil : newValue }
```

Comment:
This file is part of codegen and must be not be modified manually.

Reply:
can you point me to the command which generates it? I was adding methods to this class before manually.